frontend/src/page/Fader/FaderDistributor.tsx

```diff
@@ -1,23 +1,32 @@
 import React from "react";
 import {FaderItem} from "../../model/BackendConnection";
 import {Fader} from "./Fader";
-import {ColorFader} from "./ColorFader";
+import {HueFader} from "./HueFader";
 import {KelvinFader} from "./KelvinFader";
+import {RGBFader} from "./RGBFader";
+
 
 
 interface PropsFaderDistributor {
-    faderItem: FaderItem
+    faderItem: FaderItem,
+    setRGBItem: Function
```

Comment:
I would define the exact function type (ParameterTypes) => resultType

Reply:
Das ist mein useState setter, verstehe nicht ganz was du meinst...

Reply:
ja genau, der Setter das ist ja auch nur eine funtion. Also könntest du als Typ für diese Funktion nicht nur "function" nehmen sondern auch noch die parameter und den Rückgabewert typisieren. z.B.: (item: RGBItem) => void

Reply:
Ach so, OK Mache ich